config/samples/backends/bases/lvm/lvm.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: master
+    machineconfiguration.openshift.io/role: worker
     service: cinder
     component: cinder-volume
```

Comment:
Independent of this PR (which looks fine), we'll need to ensure this machineConfig is applied to only one worker, and the associated c-vol service is constrained to that worker.